src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.Impl.cs

```diff
@@ -47,6 +47,23 @@ Paint IView.Background
 
 		IShape IView.Clip => Clip;
 
+		IShadow IView.Shadow => Shadow;
+
+		public static readonly BindableProperty ShadowProperty =
+ 			BindableProperty.Create(nameof(Shadow), typeof(Shadow), typeof(VisualElement), defaultValue: null, propertyChanged: (b, o, n) =>
+			 {
+				 if (n is Shadow shadow)
+				 {
+					 shadow.PropertyChanged += (s, e) => (b as VisualElement).Handler?.UpdateValue(nameof(Shadow));
```

Comment:
probably need to unsubscribe from o as well?

Reply:
Done